README.md

```diff
@@ -59,77 +59,73 @@ ika --version
 ### Working with GitHub Projects
 
 ```bash
-ika add-pull-request [--column-url] [--pull-request-url]
+ika projects add [--column-url] [--url]
 
 # Add a pull request to a GitHub project column.
 ```
 
 ```bash
-ika close-project <github-url>
+ika projects close <github-url>
 
 # Set the state of an existing project board to `closed`.
 ```
 
 ```bash
-ika create-project <github-url> [--body]
+ika projects create <github-url> [--body]
 
 # Create a new project.
 ```
 
-@see: [`src/commands/projects/README.md`](https://github.com/Financial-Times/ika/blob/master/src/commands/projects/README.md) for more details.
-
 ### Working with GitHub Pull Requests
```

Comment:
Could you add documentation for `ika issues` too?

Reply:
Most certainly.